src/com/google/javascript/jscomp/GatherModuleMetadata.java

```diff
@@ -213,6 +213,9 @@ public boolean shouldTraverse(NodeTraversal t, Node n, Node parent) {
         case MODULE_BODY:
           currentModule.hasModuleBody = true;
           break;
+        case DYNAMIC_IMPORT:
```

Comment:
Could you add a test case to cover this to `GatherModuleMetadataTest`?

Reply:
Done